### PR TITLE
added edit functionality

### PR DESCRIPTION
**Pull request by UmaySoyluer**

### CODE REVIEW

File: components/Form.js [hunk 1]
```diff
@@ -1,8 +1,92 @@
...
+
   return (
-    <form>
-      <label htmlFor=""></label>
-      <input type="text"></input>
-    </form>
+    <>
+      <form onSubmit={handleSubmit}>
+        <label htmlFor="title">
+          {" "}
```
Comment (fel-sad) [inline components/Form.js:19]:
```suggestion
```

File: components/Form.js [hunk 2]
```diff
@@ -1,8 +1,92 @@
...
+          <input
+            type="text"
+            name="title"
+            id="title"
+            required
+            defaultValue={value?.title}
+          ></input>
+        </label>
+        <label htmlFor="description">
+          {" "}
```
Comment (fel-sad) [inline components/Form.js:30]:
```suggestion
```

File: components/Form.js [hunk 3]
```diff
@@ -1,8 +1,92 @@
...
+          <textarea
+            type="text"
+            name="description"
+            id="description"
+            required
+            defaultValue={value?.description}
+          ></textarea>
+        </label>
+        <label htmlFor="startDate">
+          {" "}
```
Comment (fel-sad) [inline components/Form.js:41]:
```suggestion
```

File: components/Form.js [hunk 4]
```diff
@@ -1,8 +1,92 @@
...
+          <input
+            type="date"
+            name="startDate"
+            id="startDate"
+            required
+            defaultValue={value?.startDate}
+          ></input>
+        </label>
+        <label htmlFor="endDate">
+          {" "}
```
Comment (fel-sad) [inline components/Form.js:52]:
```suggestion
```

File: components/Form.js [hunk 5]
```diff
@@ -1,8 +1,92 @@
...
+          <input
+            type="date"
+            name="endDate"
+            id="endDate"
+            required
+            defaultValue={value?.endDate}
+          ></input>
+        </label>
+        <label htmlFor="department">
+          {" "}
```
Comment (fel-sad) [inline components/Form.js:63]:
```suggestion
```

File: components/Form.js [hunk 6]
```diff
@@ -1,8 +1,92 @@
...
+            required
+            defaultValue={value?.department}
+          >
+            <option value="development">Development</option>
+            <option value="design">Design</option>
+            <option value="marketing">Marketing</option>
+          </select>
+        </label>
+        <label htmlFor="teamLead">
+          {" "}
```
Comment (fel-sad) [inline components/Form.js:77]:
```suggestion
```

File: components/Form.js [hunk 7]
```diff
@@ -1,8 +1,92 @@
...
+          Project Lead
+          <textarea
+            type="text"
+            name="teamLead"
+            id="teamLead"
+            required
+            defaultValue={value?.teamLead}
+          ></textarea>
+        </label>
```
Comment (fel-sad) [inline components/Form.js:86]: suggestion: will team leads and project leads always be the same? Project lead could be the better naming for this. :)

**Review by fel-sad — APPROVED** [review]
looking good - only had some minor suggestions. :)

File: components/Form.js (hunk 7 above)
Comment (UmaySoyluer) [reply components/Form.js:86]: Thank you Felix :) you are right but we have this one in our database as team lead , so for now it should be stayed so.

**Pull request merged**